src/components/molecules/MailchimpSignupForm/CustomForm.jsx

```
@@ -34,32 +34,97 @@ const FormRow = styled.div`
+        {status === 'sending' && isTimeout && (
+          <ErrorMessage>
+            Error: Timeout - Your browser might be blocking Mailchimp.{' '}
+            <FallbackLink href={getDirectSignupUrl()} target="_blank" rel="noopener noreferrer">
+              Click here to subscribe directly
+            </FallbackLink>
+          </ErrorMessage>
+        )}
         {status === 'error' && (
-          <div dangerouslySetInnerHTML={{ __html: message }} />
+          <ErrorMessage>
+            <div dangerouslySetInnerHTML={{ __html: message }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)